src/HealthGPS.Input/model_parser.cpp

```diff
@@ -22,6 +24,56 @@
 #define MEASURE_FUNCTION()
 #endif
 
+namespace {
+// The latest schema version for each model
+constexpr int DummyModelSchemaVersion = 1;
+constexpr int EBHLMModelSchemaVersion = 1;
+constexpr int KevinHallModelSchemaVersion = 1;
+constexpr int HLMModelSchemaVersion = 1;
+constexpr int StaticLinearModelSchemaVersion = 2;
```

Comment:
No need to worry about it for now, but will it be possible eventually to load older configs using older schemas? For instance by using the config's schema version directly?

Reply:
Yep, that's the plan. Atm I'm not requiring the `$schema` attribute, but we could do. I was just thinking that we could try to validate a file with the newest version of the schema and if that doesn't work, we try the next oldest etc.